### DISCUSSION TITLE
not compatible with the current version of VS Code (version 1.55.0)

### DISCUSSION THREAD

**Discussion by MarcelXia** [community/community#7333] [Copilot Conversations] [answered]
Unable to install 'github.copilot' extension because it is not compatible with the current version of VS Code (version 1.55.0).

How can this be, since this is the latest?

**Comment by Jollywatt** [comment]
I have the same issue, with VS Code on a university-administered computer. No such problem on my personal computer. Perhaps it is do to with licensing?

**Comment by unllamas** [comment]
Same problem: `Unable to install 'github.copilot' extension because it is not compatible with the current version of VS Code (version 1.56.2)`. Any ideas?

- MacBook Pro (Mid 2015)
- VS Code version 1.56.2

**Comment by BraedenP232** [comment]
I had the same issue as well: **Unable to install 'github.copilot' extension because it is not compatible with the current version of VS Code (version 1.55.2)**.
- MacBook Pro 2020

I was able to fix it! I followed this [fix](https://github.com/microsoft/vscode/issues/7426#issuecomment-425093469).


Workaround 1: Move Code out of Downloads and into the Applications folder.

Workaround 2: If updates still don't work after moving Code to the Applications folder, you can fix it with xattr:

sudo chown $USER ~/Library/Caches/com.microsoft.VSCode.ShipIt/*
xattr -dr com.apple.quarantine /Applications/Visual\ Studio\ Code.app


Worked for me, hope it works for other people.

**Comment by NFTMerchant** [comment]
Same issue.  Anyone find a way to install it?

**Comment by davecheney** [comment, marked as answer]
The latest version of vscode is 1.68, please upgrade.
